container-host-files/opt/scf/bin/klog.sh

```diff
@@ -21,7 +21,13 @@ if [ "$1" == "-f" ]; then
   FORCE=1
 fi
 
-NS=${1-scf}
+# read namespace val from args if provided
```

Comment:
Not sure I follow this. `NS` could come either from `$1` or `$2`?

Reply:
Apparently yes. In Jenkins he is calling it with `klog -f NS` ...
It might be easier to handle things with `shift` to squash the processed argument(s) before going to the next.

Reply:
I had a log at the entire section now, and now I am confused too.
The `-f` handling does a shift. I have now no idea what this construction is for either.

Reply:
Yes. https://github.com/SUSE/scf/blob/fec8cd4284c9cbc43bb0775519b2dd9599e6aded/container-host-files/opt/scf/bin/klog.sh#L20

Reply:
Agreed. The code seems redundant. 

`NS=${1-scf}` this bit enough